.changeset/beige-lamps-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-msgraph': patch
```

Comment:
should be minor since it's a breaking change to a public interface

Reply:
I guess you mean the change to the client.getUsers and client.getGroups functions, right @freben? Besides that queryMode is optional with backwards compatible default value applied at a low level.

If it is about these two functions, maybe a change of the position would reduce it again to "patch"? 🤔

Reply:
I think it would, yes

Reply:
@freben I moved the new optional parameter to the end